scripts/jest/utils.js

```diff
@@ -0,0 +1,21 @@
+import { JSDOM, VirtualConsole } from 'jsdom';
+import fs from 'fs';
+
+exports.createJSDOM = done => {

```

Comment:
Does this work? I was trying to do something similar because the repetition was annoying me, but kept hitting a brick wall. I just tried locally again fiddling around with this branch instead but had similar results.
I can't see this currently being used or hooked up to anything, unless there's a plan for it?